Hazel/vendor/Glad/src/glad.c

```diff
@@ -1,11 +1,11 @@
 /*
```

Comment:
Same question, What are you accomplishing by doing this? These are machine generated files and should be generally be left as untouched.

Reply:
Same question in same module as [this comment](https://github.com/TheCherno/Hazel/pull/82/commits/0853563d568f55f7449b63b50f4ab48c23c708c2#r308013630)